library/bignum_mod_raw.h

```diff
@@ -33,6 +33,59 @@
 
 #include "bignum_mod.h"
 
+/**
+ * \brief   Perform a safe conditional copy of MPI which doesn't reveal whether
+ *          the condition was true or not.
+ *
+ * The size to copy is determined by \p N.
+ *
+ * \param[out] X        The address of the first MPI. This must be initialized.
+ *                      Must have enough limbs to store the full value of \p A.
+ * \param[in]  A        The address of the second MPI. This must be initialized.
+ * \param[in]  N        The address of the modulus related to \p X and \p A.
+ * \param      assign   The condition deciding whether to perform the
+ *                      assignment or not. Must be either 0 or 1:
+ *                      * \c 1: Perform the assignment `X = A`.
+ *                      * \c 0: Keep the original value of \p X.
+ *
+ * \note           This function avoids leaking any information about whether
+ *                 the assignment was done or not.
+ *
+ * \warning        If \p assign is neither 0 nor 1, the result of this function
+ *                 is indeterminate, and the resulting value in \p X might be
+ *                 neither its original value nor the value in \p B.
+ */
+void mbedtls_mpi_mod_raw_cond_assign( mbedtls_mpi_uint *X,
+                                      const mbedtls_mpi_uint *A,
+                                      const mbedtls_mpi_mod_modulus *N,
+                                      unsigned char assign );
+
+/**
+ * \brief   Perform a safe conditional swap of MPI which doesn't reveal whether
```

Comment:
missing word before `MPI`
```suggestion
 * \brief   Perform a safe conditional swap of two MPIs which doesn't reveal whether
```